sds_data_manager/lambda_code/SDSCode/instruments/l1b_Codice.py

```diff
@@ -0,0 +1,37 @@
+"""Lambda runtime code that triggers off of arrival of data into S3 bucket.
+"""
+import os
+from datetime import datetime
+
+import boto3
+
+#TODO: ability to access database, EFS, calibration data, etc.
+
+def lambda_handler(event: dict, context):
+    """Handler function"""
+    print(event)
+    print(context)
+
+    now = datetime.now()
+    print("Now time is")
+    print(now)
+
+    # Get the environment variables
+    bucket = os.environ['PROCESSING_PATH']
+    prefix = os.environ['INSTRUMENT_SOURCES']
+
+    # Retrieves objects in the S3 bucket under the given prefix
+    try:
+        s3 = boto3.client('s3')
+        object_list = s3.list_objects_v2(Bucket=bucket, Prefix=prefix)["Contents"]
+        print(object_list)
+    except KeyError:
+        print("No files present.")
```

Comment:
Should this alter our STATE below? If nothing found: FAIL.

I see, below you do add that as a TODO.